packages/webpack-plugin/src/webpack-config-stylable-excludes.ts

```diff
@@ -54,3 +54,31 @@ function safelyWalkJSON(
         }
     }
 }
+
+export function bundleLibs(config: Configuration, packages: string[], isServer: boolean) {
+    if (isServer) {
+        if (Array.isArray(config.externals) && config.externals.length === 1) {
+            const nextExternal = config.externals[0];
+
+            if (typeof nextExternal === 'function') {
+                config.externals = [
+                    async (ctx: any) => {
+                        for (const pack of packages) {
+                            if (ctx.request.startsWith(pack)) {
+                                return false;
+                            }
+                        }
+                        return nextExternal(ctx, (data) => {
+                            (data as any).resolve();
```

Comment:
`data.resolve()`? - shouldn't this function be called with `err, result, type?` - https://webpack.js.org/configuration/externals/#function

Why not pass in the 2nd argument callback that the wrapper function takes?

Reply:
fixed